Formula/cargo-outdated.rb

```diff
@@ -16,9 +16,9 @@ class CargoOutdated < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "30bc7667a6e1bf3e04835e0db2426b25d2bbbc0290140d75ec7200b85f3e002e"
   end
 
+  depends_on "rust" => [:build, :test]
```

Comment:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => [:build, :test]
```

Reply:
Done.

Reply:
@carlocab There's still a warning when building on macOS 11 ARM:

```
Full linkage --cached --test --strict cargo-outdated output
  Undeclared dependencies with linkage:
    openssl@3
```

Is there anything to worry about here? Thanks!

Reply:
Yes, this will break things. Let me have a look...

Reply:
Try
```diff
diff --git a/Formula/cargo-outdated.rb b/Formula/cargo-outdated.rb
index 0689dd562ae..428a9e54158 100644
--- a/Formula/cargo-outdated.rb
+++ b/Formula/cargo-outdated.rb
@@ -21,6 +21,8 @@ class CargoOutdated < Formula
   depends_on "rust"
 
   def install
+    ENV["OPENSSL_NO_VENDOR"] = "1"
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
     system "cargo", "install", *std_cargo_args
   end
```

Reply:
Done.